UnitTests/GitCommandsTests/GitModuleTest.cs

```diff
@@ -534,9 +538,12 @@ public void GetSuperprojectCurrentCheckout()
                 moduleTestHelperSuper.Module.GitExecutable.GetOutput(@"commit -am ""Update submodule ref""");
 
                 // Assert
-                (char code, ObjectId commitId) = moduleSub.GetSuperprojectCurrentCheckout();
-                Assert.AreEqual(32, code);
-                Assert.AreEqual(commitRef, commitId.ToString());
+                ThreadHelper.JoinableTaskFactory.Run(async () =>
```

Comment:
💭 Why not just make the containing test method asynchronous?

Reply:
I don't know. Is doing so compatible with JTF?